python/foglamp/services/core/api/auth.py

```diff
@@ -62,35 +64,61 @@
         curl -X POST -d '{"username": "user", "password": "foglamp"}' http://localhost:8081/foglamp/login
     """
 
-    data = await request.json()
-    username = data.get('username')
-    password = data.get('password')
+    data = await request.text()
+    if data.startswith("-----BEGIN CERTIFICATE-----"):
```

Comment:
Add `&& data.endswith('-----END CERTIFICATE-----')`
Is it better check to deal with?

Reply:
We can add but will it not be an overkill?

Reply:
agree with @amarendra-dianomic!  we just want to know login is with certificate? and verification is being done for certificate